pages/linux/pacgraph.md

```diff
@@ -0,0 +1,36 @@
+# pacgraph
+
+> Draw a graph of installed packages to PNG/SVG/GUI/console. Good for finding bloat.
+> More information: <https://github.com/keenerd/pacgraph>.
+
+- Produce SVG and PNG graph:
+
+`pacgraph`
+
+- Produce SVG graph:
```

Comment:
```suggestion
- Produce a SVG graph:
```

Reply:
Can you change the `a` articles to `an`?

Reply:
Done